packages/vue-app/package.json

```diff
@@ -21,5 +21,9 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
```

Comment:
I suggest to keep dev dependencies in the root pkg

Reply:
:ok: